### ISSUE TITLE
the trait `AsRef<candle_core::tensor::Tensor>` is not implemented for `&candle_core::Tensor`

### ISSUE THREAD

**Issue by jk2K** [tomsanbear/candle-einops#3]
my code
```rust
#[test]
fn test_einops_repeat() -> Result<(), candle_core::Error> {
    // let a1 = Tensor::new(&[
    //     [1., 2., 3.],
    //     [2., 4., 6.]
    // ], &Device::Cpu).unwrap();
    let a1 = Tensor::arange(0f32, (10 * 20 * 30 * 40) as f32, &Device::Cpu)?.reshape(&[10, 20, 30, 40]).unwrap();
    let repeat = 5;
    let output = einops!("h w c -> h {repeat} w c", a1);

    Ok(())
}
```

get error
```
error[E0277]: the trait bound `&candle_core::Tensor: Backend` is not satisfied
  --> tests/tensor_tests.rs:57:18
   |
57 |     let output = einops!("h w c -> h {repeat} w c", a1);
   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `AsRef<candle_core::tensor::Tensor>` is not implemented for `&candle_core::Tensor`, which is required by `&candle_core::Tensor: Backend`
   |
   = help: the trait `AsRef<candle_core::Tensor>` is implemented for `candle_core::Tensor`
   = help: for that trait implementation, expected `candle_core::Tensor`, found `&candle_core::Tensor`
   = note: required for `&candle_core::Tensor` to implement `AsRef<candle_core::tensor::Tensor>`
   = note: required for `&candle_core::Tensor` to implement `Backend`
   = note: this error originates in the macro `einops` (in Nightly builds, run with -Z macro-backtrace for more info)
```

why is this? what should I do ? thanks

candle core version: 0.6.0
candle-einops version: 0.1.1

**Comment by jk2K** [comment]
`candle-einops` need to release new version, currently 0.1.1 depends on candle core 0.4, there are compatibility issues

**Comment by tomsanbear** [comment]
Just published version 0.1.2 with candle 0.6 as the dependency, feel free to reopen if any other issues appear 👍
